server/davidia/server/processor.py

```diff
@@ -29,7 +29,8 @@
 def check_line_names(lines: list[LineData]) -> list[LineData]:
     """Autonames lines that do not have names"""
     used_names = set(line.line_params.name for line in lines if line.line_params.name)
-    for index, line in enumerate(lines):
+    index = 0
+    for line in lines:
         if not line.line_params.name:
             new_name = f"Line {index}"
```

Comment:
increment `index` after use otherwise you match again for next line with no name